states/battle.py

```diff
@@ -2,6 +2,7 @@
 
 
 class Battle:
+    """Will handle the "Battle" function"""
```

Comment:
To be specific this is a `state`. It's in the `states` folder and is one of the possible game `states`. Maybe something like 'Game state for player battling' or something and make sure whatever you choose, the other game states' comments are also consistent with this one

Reply:
Changed to `Game state for player battling`